.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+version: 2.1
+tagged_build_filters: &tagged_build_filters
+  branches:
+    ignore: /.*/
+  tags:
+    only: /v[0-9]+\.[0-9]+\.[0-9]+/
+test_build_filters: &test_build_filters
+  branches:
+    only: /.*/
+  tags:
+    ignore: /v[0-9]+\.[0-9]+\.[0-9]+/
+jobs:
+  test:
+    docker:
+      - image: circleci/ruby:2.6.5
+    steps:
+      - checkout
+      - run: bundle install
+      - run: rubocop -c .rubocop.yml
+      - run: rake test
+  build:
+    docker:
+      - image: circleci/ruby:2.6.5
+    steps:
+      - checkout
+      - run:
+          name: Check Tagged Push
+          command: |
+            PKG_VERSION=$(grep VERSION lib/logdna/version.rb | cut -d'"' -f2)
```

Comment:
I don't think this file exists yet; believe the version is currently hardcoded in the `.gemspec`.

Reply:
nice catch, @matt-march, updated!